### DISCUSSION TITLE
Support debug log output

### DISCUSSION THREAD

**Discussion by poday** [rjmacarthy/twinny#101] [Ideas]
I ran into an issue, ollama needed to be restarted, and I was looking for twinny's log output to understand what was going on. However I'm not seeing the twinny in the dropdown for plugin log output. Adding some log statements would provide insight into how the plugin is behaving and help diagnosing problems.

**Comment by rjmacarthy** [comment]
Hey, I just released a new version `3.5.3` with the option to enable debugging for streaming completions, it should help determine what is happening.

**Reply by dpkirchner** [reply]
Where are the debug logs written? I tried to find them by going to `Developer: Show Logs` -> `Extension Host` but there's nothing there (other than what appears by default, ie all the extension activation events).
